std/experimental/allocator/mmap_allocator.d

```diff
@@ -28,6 +28,7 @@ struct MmapAllocator
     version(Posix)
     {
         /// Allocator API.
+        nothrow @trusted @nogc
         void[] allocate(size_t bytes) shared
         {
             import core.sys.posix.sys.mman : mmap, MAP_ANON, PROT_READ,
```

Comment:
On a second look, it might be better to use a `@trusted` lambda around mmap as `@trusted` on functions is a really anti-pattern that we soon might automatically check against.

Reply:
Can we make make `mmap` and `VirtualAlloc` pure?